src/styles/index.scss

```diff
@@ -1,3 +1,13 @@
+@import 'main';
+
 body {
   margin: 0;
```

Comment:
There is a redundancy in the `body`'s margin properties. You have set `margin: 0;` on line 4 and then overridden it with `margin-left: 5px;` on line 12. You should consolidate these properties into a single margin declaration or remove the unnecessary margin-left.